docs/docs-components/AppLayout.js

```diff
@@ -109,11 +109,11 @@ export default function AppLayout({ children, colorScheme }: Props): ReactNode {
           <Box
             display="none"
             mdDisplay="block"
-            position="fixed"
```

Comment:
Q: am curious what is this fixing?

Reply:
Not a fix but improvement. I started testing in docs navigation first and If we ever wanted it to be collapsible, fixed position wouldn't work. Because sidenav should push the main page content when its width changes. it won't behave like that if is a fixed element.

Also It's inside a flexbox but fixed position makes it non-flex item.